### ISSUE TITLE
o-forms: save/saving state not accessible to screen reader users - ID: DAC_Status_Message_04

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami#537]
# Issue
This is from page 116 of the [audit report](https://drive.google.com/file/d/17wq6b0CIX41RA4muBFDzUDxE0-qzHBIH/view)

Issue can be found on: <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=radio-box&system_code=origami-repo-data&brand=master>

The radio buttons with the inline saved state may be problematic for screen reader users as the content which displays the ‘saved’ or ‘saving’ states has been added using CSS. As there is no text alternative, screen reader users would be unable to view the state of the buttons in the same way as a sighted user.

 
# Proposed Solution:
Ensure that there is an alternative available for screen reader users such as a status message. Further information can be found in [Using role=status to present status messages](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22).

**Comment by notlee** [comment]
More info here: https://www.w3.org/TR/WCAG20-TECHS/F87.html
